optimum/exporters/onnx/__main__.py

```diff
@@ -116,7 +116,12 @@ def main():
         args.for_ort and (model.config.is_encoder_decoder or task.startswith("causal-lm"))
     ):
         if task == "stable-diffusion":
-            output_names = ["text_encoder/model.onnx", "unet/model.onnx", "vae_decoder/model.onnx"]
+            output_names = [
+                "text_encoder/model.onnx",
```

Comment:
Is it always necessary? For example, `pipeline_stable_diffusion.py` has no call to `encode()`, while `pipeline_cycle_diffusion.py` and `pipeline_stable_diffusion_img2img.py` do.